File: 01-Front/Saniteau/client-src/model/compteur/EnregistreCompteurRequest.ts
export class EnregistreCompteurRequest {
    public idCompteur: number;
    public nomCompteur: string;


    constructor(idCompteur: number, nomCompteur: string) {
        this.idCompteur = idCompteur;
        this.nomCompteur = nomCompteur;
    }

}

File: 01-Front/Saniteau/Models/Compteurs/EnregistreCompteurRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Saniteau.Models.Compteurs
{
    public class EnregistreCompteurRequest
    {
        public int IdCompteur { get; set; }
        public string NomCompteur { get; set; }
    }
}
